packages/create-codes/README.md

```diff
@@ -1 +1,5 @@
 # create-codes
+
+## Getting Started
+
+To develop locally, You can use the `pnpm dev` command.
```

Comment:
Would it be better to provide additional information about the command? 
For example,
```
This will enable watch mode, which triggers a rebuild whenever changes are made.
```